lib/stat_tracker.rb

```diff
@@ -0,0 +1,38 @@
+require 'CSV'
+# require './lib/game.rb'
+# require './lib/team.rb'
+# require './lib/game_teams.rb'
+
+class StatTracker
+  attr_reader :games_data,
+              :teams_data,
+              :game_teams_data
+
+  def initialize(games_data, teams_data, game_teams_data)
+    @games_data = games_data
+    @teams_data = teams_data
+    @game_teams_data = game_teams_data
+  end
+
+  def self.from_csv(stats_params)
+    games_data = []
+    CSV.foreach(stats_params[:games], headers: true, header_converters: :symbol) do |row|
+      games_params = row.to_hash
+      games_data << games_params
```

Comment:
Once the Game class has been created, this line will need to be changed to `games_data << Game.new(games_params)`

Reply:
Game class set up in alternate branch to accept params hash.